bgpd/bgp_routemap.c

```diff
@@ -4167,14 +4171,18 @@ static const char *parse_evpn_rt_type(const char *num_rt_type)
 
 DEFUN_YANG (match_evpn_route_type,
 	    match_evpn_route_type_cmd,
-	    "match evpn route-type <macip|2|multicast|3|prefix|5>",
+	    "match evpn route-type <1|ead|macip|2|multicast|3|es|4|prefix|5>",
 	    MATCH_STR
 	    EVPN_HELP_STR
 	    EVPN_TYPE_HELP_STR
+            EVPN_TYPE_1_HELP_STR
```

Comment:
Does this seem a bad indentation, isn't it?

Reply:
ack. Fixed it.